src/use_case/choose_genre/ChooseInteractor.java

```diff
@@ -11,7 +11,7 @@ public ChooseInteractor(ChooseOutputBoundary genrePresenter, ChooseDataAccessInt
         this.genreDataAccessObject = genreDataAccessObject;
     }
     public void execute(ChooseInputData chooseGenreInputData){
-        ChooseOutputData outputData = new ChooseOutputData(chooseGenreInputData.getGenre());
+        /*ChooseOutputData outputData = new ChooseOutputData(chooseGenreInputData.getGenre());
```

Comment:
just curious why are we commenting this entire section out? is this going to be deleted or something like that?